bots/pr/src/main/java/org/openjdk/skara/bots/pr/JEPCommand.java

```diff
@@ -79,6 +79,11 @@ private Optional<Issue> getJepIssue(String args, PullRequestBot bot) {
     @Override
     public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath, CommandInvocation command,
                        List<Comment> allComments, PrintWriter reply, List<String> labelsToAdd, List<String> labelsToRemove) {
+        if (!bot.enableJep()) {
+            reply.println("this repository is not allowed to use the `jep` command.");
```

Comment:
```suggestion
            reply.println("This repository has not been configured to use the `jep` command.");
```

Reply:
Does the csr command also use the `has not been configured`.

Reply:
Fixed. And the csr command is also adjusted.